Coding/fw/src/Adapters/CANAdapter.h

```diff
@@ -58,6 +58,7 @@ class CANAdapter : public CANInterface
     bool begin()
     {
         bool success = true;
+        m_Can_Controller.setPins(5, 4);
```

Comment:
magic numbers, either use  const variables with descriptive names, or add comment what you do here,